src/signals/incident-management/containers/Dashboard/components/ComparisonRate.tsx

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import { Description, ComparisonRateWrapper as Wrapper } from './styled'
+
+interface Props {
+  percentage: number
+}
+
+export const ComparisonRate = ({ percentage }: Props) => {
+  const direction = percentage < 0 ? 'down' : 'up'
+
+  return (
+    <Wrapper>
+      <span>
+        {
+          <img
+            src={`/assets/images/dashboard/arrow-${direction}.svg`}
```

Comment:
Kan aan mij liggen, maar ik zie dit helemaal niet terug in de grafiek.

Reply:
<img width="575" alt="Screenshot 2023-02-01 at 12 23 58" src="https://user-images.githubusercontent.com/46756331/216029692-4a910e10-1b14-422c-94a4-e4859537c57a.png">